KEGG_parser/parsers.py

```diff
@@ -22,30 +64,81 @@ def split_and_append(current_dict, current_entry_name, current_entry_data):
         current_dict[current_entry_name] = list()
 
     current_dict[current_entry_name].append((current_entry_pathway_id, current_entry_pathway_name))
+
+    return current_dict
+
+def split_and_append_organism(current_dict, current_entry_name, current_entry_data):
+    split_current_entry_data = current_entry_data.split()
+    current_entry_pathway_id = split_current_entry_data[0]
+    current_entry_pathway_name = ' '.join(split_current_entry_data[1:])
+    current_dict[current_entry_name] = (current_entry_pathway_id, current_entry_pathway_name)
+
     return current_dict
 
 def add_class(current_dict, current_entry_name, current_entry_data):
     if current_entry_name in current_dict:
         current_dict[current_entry_name].append(current_entry_data)
     else:
         current_dict[current_entry_name] = [current_entry_data]
+
     return current_dict
 
 def add_nested_dict(current_dict, current_entry_name, current_entry_data):
```

Comment:
@shafferm The logic in this differed in each of the parsers (line 90 was indented) but I don't think this makes a difference but just might've been a silent bug. Let me know if this fix makes sense